router/build.gradle

```diff
@@ -58,8 +58,8 @@ afterEvaluate {
                 from components.release
 
                 group = "com.github.zsoltk"
-                artifactId = "final"
-                version = "0.24.0"
+                artifactId = "compose-router"
+                version = "0.24.1"
```

Comment:
Does this have to be maintained to work? So far this part was automatic. I'm thinking this is potentially easy to forget about. But I guess it doesn't work without this for some reason?

Reply:
Not sure actually, working with publication for the first time as well
@AbdElraoufSabri could you please give a hint?

Reply:
UPD: yes, it works well without these parameters. Will remove them and push a commit

Reply:
Sure, Those are for the maven central plugin I think not the jitpack. Really was thinking of providing the library through maven central. I can provide it under my namespace for example `tech.abd3lraouf:compose-router:x.y.z`.

Really was thinking of creating the library with dynamic versioning from commit messages, I've little exp with it but I maybe migrate all my open source libraries to use this approach.

Reply:
If it does work without them then the only difference is a newline at the end of `jitpack.yml`? I need to test that out, big if true.

Reply:
Still fails. https://jitpack.io/com/github/zsoltk/compose-router/4bc8250/build.log 

What's your secret sauce? :) Maybe the other changes are needed after all.

Reply:
No, I meant that you don't need to explicitly specify `groupId`, `artifactId` and `version`. Applying the `maven-publish` and creating a `release(MavenPublication)` publication is still required :D